templates/zerver/integrations/jitsi.md

```diff
@@ -0,0 +1,16 @@
+Zulip integrates with [Jitsi Meet](https://jitsi.org/jitsi-meet/),
+by default for video chat support. It is fully-encrypted, 100% open
+source video conferencing solution. Jitsi Meet is supported by
+default so no extra configuration is required to use it.
+
+### Using Jitsi Meet
+
+1. **Set jitsi as Video chat provider in Organization settings**.
```

Comment:
For this first step, I'd link to a /help/ article for "Set video chat provider".  Which actually is probably where the "disable video chat provider" link should go as well.

Reply:
We currently do not have a `/help` article for "Set video chat provider", there is a section `Change your video chat provider` which now points to these different providers in respective `/integrations` page. Should I add one for "Set video chat provider".